addons/l10n_in_edi_ewaybill/models/ewaybill_type.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class EWayBillType(models.Model):
+    _name = "l10n.in.ewaybill.type"
+    _description = "E-Waybill Document Type"
+
+    name = fields.Char("Document Type")
+    code = fields.Char("Code")
+    allowed_in_supply_type = fields.Selection(
+        [
+            ("both", "Incoming and Outgoing"),
+            ("out", "Outgoing"),
+            ("in", "Incoming"),
+        ],
+        string="Allowed in supply type",
+    )
+    active = fields.Boolean("Active", default=True)
+    parent_type_ids = fields.Many2many(
```

Comment:
What is this complication good for?

Reply:
because of this https://docs.ewaybillgst.gov.in/apidocs/sub-docType-mapping.html